src/components/logos/LogoSonar.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import { forwardRef } from 'react';
+import { useIntl } from 'react-intl';
+
+interface Props {
+  hasText?: boolean;
+}
+
+export const LogoSonar = forwardRef<SVGSVGElement, Props>(({ hasText = false, ...props }, ref) => {
+  const intl = useIntl();
+
+  return hasText ? (
+    <svg
+      fill="none"
+      height="48"
+      ref={ref}
+      role="img"
+      viewBox="0 0 152 48"
+      width="152"
```

Comment:
Could having the height and width hardcoded like this prevent us to use the logos to just fill some available space? or just passing `height="100%" width="100%"` would work fine?

Reply:
Hmm, I didn't think we'd have variable sizes on these. I'll check with Marcio